docker-compose.dev.yml

```diff
@@ -29,7 +29,7 @@ services:
   theme_classifier:
     image: "openstatefoundation/ori-theme-classifier:v1.0.0"
     ports:
-      - "80:5000"
+      - "82:5000"
```

Comment:
Please revert this

Reply:
Can I ask why? I changed this for the local dev version because it can conflict with other services trying to use the default internet port.

Reply:
I agree that port 80 here is not ideal, it makes more sense to use 5000:5000 if available. However, be aware that env vars might need to change as well.
Normally I don't commit my local port adjustments cause they could be conflicting for another person.

Reply:
It's been reverted.